zinit-autoload.zsh

```diff
@@ -1413,70 +1413,74 @@ EOF
+  bindkeys         Lists bindkeys set up by each plugin
+  cd               Change directory to a plugin or snippet
+  cdclear          Clear compdef replay list
+  cdisable         Disable completion
+  cdlist           Show compdef replay list
+  cdreplay         Replay compdefs
+  cenable          Enable completion
+  changes          View plugin's git log
+  compile          Compile plugin
+  compiled         List plugins that are compiled
+  compinit         Refresh installed completions
+  completions      List completions in use
+  create           Create plugin (also together with Github repository)
+  creinstall       Install a plugin or local completion
+  csearch          Search for available completions from any plugin
+  cuninstall       Uninstall completions for plugin
+  debug            Manage debug mode
+  delete           Delete a plugin or snippet
+  edit             Edit plugin's file with \$EDITOR
+  glance           Look at plugin's source
+  help             Usage information
+  ice              Add ICE to next command
+  light            Light plugin load, without reporting/tracking
+  load             Load plugin, can also receive absolute local path
+  man              Manual
+  module           Manage binary Zsh module shipped with Zinit
+  plugins          List currently installed plugins
+  recall           Fetch saved ice modifiers and construct 'zinit ice ...' command
+  recently         Show plugins that changed recently
+  report           Show plugin's report
+  self-update      Update zinit
+  snippet          Source local or remote file (by direct URL)
+  snippets         List currently installed snippets
+  status           Git status for plugin or svn status for snippet
+  stress           Test plugin for compatibility with set of options
+  times            Statistics on plugin load times, sorted in order of loading
+  uncompile        Remove compiled version of plugin
+  unload           Unload plugin
+  update           Update a plugin or snippet
+  version          Display zinit version
+  zstatus          Overall zinit status
+EOF
+
+integer idx
+local type key
+local -a arr
+for type in subcommand hook; do
+    for (( idx=1; idx <= ZINIT_EXTS[seqno]; ++ idx )); do
+        key="${(k)ZINIT_EXTS[(r)$idx *]}"
+        [[ -z "$key" || "$key" != "z-annex $type:"* ]] && continue
+        arr=( "${(Q)${(z@)ZINIT_EXTS[$key]}[@]}" )
+        (( ${+functions[${arr[6]}]} )) && { "${arr[6]}"; ((1)); } || \
+            { builtin print -rl -- "(Couldn't find the help-handler \`${arr[6]}' of the z-annex \`${arr[3]}')"; }
     done
+done
+
+cat <<EOF
+Run 'zinit COMMAND --help' for more information on a command.
 
-local -a ice_order
-ice_order=( ${${(s.|.)ZINIT[ice-list]}:#teleid} ${(@)${(@)${(@Akons:|:u)${ZINIT_EXTS[ice-mods]//\'\'/}}/(#s)<->-/}:#(.*|dynamic-unscope)} )
-print -- "\nAvailable ice-modifiers:\n\n${ice_order[*]}"
+For more help on how to use Ziniit, head to https://github.com/zdharma_continuum/zinit'.
```

Comment:
```suggestion
For more help on how to use Zinit, head to https://github.com/zdharma_continuum/zinit'.
```

Reply:
Good callout. I updated PR to use `zinit` & `zsh`  after looking at the `zsh` man page.